internal/mount/flag.go

```diff
@@ -96,3 +116,26 @@ func MetadataCacheTTL(statCacheTTL, typeCacheTTL time.Duration, ttlInSeconds int
 
 	return
 }
+
+// StatCacheMaxSizeMB returns the stat-cache size in MiBs based on the user old and new flags/configs.
+func StatCacheMaxSizeMB(mountConfigStatCacheMaxSizeMB int64, flagStatCacheCapacity int) (statCacheMaxSizeMB uint64, err error) {
```

Comment:
it should be named something like ResolveStatCacheMaxSizeMB ?

Reply:
done